### ISSUE TITLE
Clip by global norm, not local norm

### ISSUE THREAD

**Issue by hoangcuong2011** [keras-team/keras#13575]
Greetings,

I noticed clipnorm in Keras is with [clip_by_local_norm](https://www.tensorflow.org/api_docs/python/tf/clip_by_norm), and not clip_by_global_norm. However, clip_by_global_norm is recommended as it is "the correct way to perform gradient clipping" https://www.tensorflow.org/api_docs/python/tf/clip_by_global_norm

Do you think this is important? I don't think it is hard to change to clip_by_global_norm and I can work on this. But I am not sure whether the Keras community wants this or not.

**Comment by swghosh** [comment]
https://github.com/tensorflow/tensorflow/issues/36001#issuecomment-595045357
